webserver/views/api/v1/core.py

```diff
@@ -109,3 +111,20 @@ def _validate_offset(offset):
     else:
         offset = 0
     return offset
+
+@bp_core.route("/<user_api_key>/get_pending_jobs", methods=["GET"])
+def get_pending_jobs(user_api_key):
+    """ Return the pending jobs for a user.
+
+    Retrieve the Job Ids for the pending jobs for the user whose
+    user id is given as an argument and return that.
+    """
+    final_result = {}
+    try:
+        pending_jobs = db.dataset_eval.get_user_pending_jobs(user_api_key)
+        for i in range(len(pending_jobs)):
+            final_result[i+1] = str(pending_jobs[i][0])
+    except:
+        e = sys.exc_info()[0]
```

Comment:
This is quite an unusual way of getting information about the exception. Can you just use `except MyException as e`?